src/TitleForm/TitleForm.js

```diff
@@ -42,6 +43,12 @@ import ContributorsForm from './ContributorsForm';
 
 const ALLOWED_YEAR_LENGTH = 4;
 
+const validateClaimingInterval = (value, { claimingActive }) => {
```

Comment:
if the second argument is optional it's better to add the default value, otherwise `claimingActive` wouldn't be destructured
`const validateClaimingInterval = (value, { claimingActive } = {}) => {`

Reply:
Thanks @mariia-aloshyna! Relying on the interface provided by the `final-form` library, this argument is not optional and is of a type object.
https://github.com/final-form/final-form/blob/edada3668ce21f1a75bd71386464dc79f01fbb28/src/index.d.ts#L139